analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/TestHelper.cs

```diff
@@ -258,24 +258,32 @@ public static string CreateAnalysisConfig(TestContext context, IEnumerable<strin
         public static string CreateSonarProjectConfig(TestContext context, IEnumerable<string> unchangedFiles)
         {
             var analysisConfigPath = CreateAnalysisConfig(context, unchangedFiles);
-            return CreateSonarProjectConfig(Path.GetDirectoryName(analysisConfigPath), "AnalysisConfigPath", analysisConfigPath, true);
+            return CreateSonarProjectConfig(Path.GetDirectoryName(analysisConfigPath), "NotImporant", null, true, analysisConfigPath);
         }
 
+        // ToDo: Use TestContext instead of sonarProjectConfigDirectory
```

Comment:
These ToDo's are for separate PR

Reply:
...this is going to be the prerequisite PR, because it breaks stuff for existing tests